File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        
        record = {}

        for number in nums:

            if number in record:

                record[number] += 1

            else:
                record[number] = 1

        sorted_record = sorted(record.items(), key=lambda x:x[1], reverse=True)
        top_nums = []

        for i in range(k):

            top_nums.append(sorted_record[i][0])

        return top_nums

File: Data Structures & Algorithms/move-zeroes/submission-0.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {

        int num_zeros = 0;
        for(int number: nums){

            if(number == 0){
                num_zeros ++;
            }
        }


        std::erase(nums, 0);

        while(num_zeros > 0){
            nums.push_back(0);
            num_zeros --;
        }
        
    }
};